tasks/gcp/vm.py

```diff
@@ -0,0 +1,160 @@
+from typing import Optional, Tuple
+
+from invoke.context import Context
+from invoke.exceptions import Exit
+from invoke.tasks import task
+from pydantic_core._pydantic_core import ValidationError
+
+from tasks import config, doc, tool
+from tasks.config import get_full_profile_path
+from tasks.deploy import deploy
+from tasks.destroy import destroy
+from tasks.gcp import doc as gcp_doc
+from tasks.gcp.common import (
+    get_architectures,
+    get_default_architecture,
+    get_default_os_family,
+    get_deploy_job,
+    get_os_families,
+)
+from tasks.tool import clean_known_hosts as clean_known_hosts_func
+from tasks.tool import get_host, show_connection_message
+
+scenario_name = "gcp/vm"
+remote_hostname = "gcp-vm"
+
+
+@task(
+    help={
+        "config_path": doc.config_path,
+        "install_agent": doc.install_agent,
+        "install_installer": doc.install_installer,
+        "agent_version": doc.agent_version,
+        "stack_name": doc.stack_name,
+        "debug": doc.debug,
+        "interactive": doc.interactive,
+        "ssh_user": doc.ssh_user,
+        "os_family": gcp_doc.os_family,
+        "architecture": gcp_doc.architecture,
+        "instance_type": gcp_doc.instance_type,
+        "os_version": doc.os_version,
+    }
+)
+def create_vm(
+    ctx: Context,
+    config_path: Optional[str] = None,
+    stack_name: Optional[str] = None,
+    install_agent: Optional[bool] = True,
+    install_installer: Optional[bool] = False,
+    agent_version: Optional[str] = None,
+    debug: Optional[bool] = False,
+    interactive: Optional[bool] = True,
+    ssh_user: Optional[str] = None,
+    account: Optional[str] = None,
+    os_family: Optional[str] = None,
+    os_version: Optional[str] = None,
+    architecture: Optional[str] = None,
+    instance_type: Optional[str] = None,
+    deploy_job: Optional[str] = None,
```

Comment:
Is that useful? It is currently not used

Reply:
I'll drop it!